MyApp/app/models/user.rb

```diff
@@ -11,6 +11,9 @@ class User < ApplicationRecord
 
   enum role: [:admin, :author, :editor]
 
+  scope :last_seen_scope, -> { where(last_seen_at: 15.minutes.ago..DateTime.now).or(where(:last_seen_at => nil)) }
+  scope :team_scope, -> { where.not(:invitation_sent_at => nil)}
```

Comment:
@olga-kolosek team_scope filtrira rekorde gde pozivnica nije poslata? Ime treba da bude vise odgovarajuce onome sto metoda radi. To primenjivati svuda.